Formula/lib/libarchive.rb

```diff
@@ -49,7 +49,6 @@ def install
     # fixes https://github.com/libarchive/libarchive/issues/1819
     inreplace lib/"pkgconfig/libarchive.pc" do |s|
       s.gsub! "Libs.private: ", "Libs.private: -liconv "
-      s.gsub! "Requires.private: iconv", ""
     end
```

Comment:
This entire `inreplace` can be deleted, since https://github.com/libarchive/libarchive/pull/2611 fixes both https://github.com/libarchive/libarchive/issues/1766 and https://github.com/libarchive/libarchive/issues/1819. Right now the final `.pc` has two `-liconv`s:

```
Libs.private: -liconv -lexpat -llzma -lzstd -llz4 -lb2 -lbz2  -lz -liconv
```